src/grammar/next/grammar.spec.ts

```diff
@@ -9,21 +9,32 @@ expect.addSnapshotSerializer({
 });
 
 describe("grammar", () => {
-    for (const r of loadCases(__dirname + "/../test/")) {
-        it("should parse " + r.name, () => {
-            const ast = getAstFactory();
-            const { parse } = getParser(ast, "new");
-            expect(parse(r.code, "<unknown>", "user")).toMatchSnapshot();
-        });
+    const shouldParsePaths = [__dirname + "/../test/", __dirname + "/test/"];
+
+    for (const path of shouldParsePaths) {
+        for (const r of loadCases(path)) {
+            it("should parse " + r.name, () => {
+                const ast = getAstFactory();
+                const { parse } = getParser(ast, "new");
```

Comment:
this should be moved out of the loop and also reused in the new tests for the new parser